executor/aggfuncs/window_func_test.go

```diff
@@ -151,3 +177,31 @@ func (s *testSuite) TestWindowFunctions(c *C) {
 		s.testWindowFunc(c, test)
 	}
 }
+
+type windowMemTest struct {
+	windowTest         windowTest
+	allocMemDelta      int64
+	updateMemDeltaGens updateMemDeltaGens
+	isDistinct         bool
+}
+
+func buildWindowMemTester(funcName string, tp byte, constantArg uint64, orderByCols int, numRows int, allocMemDelta int64, updateMemDeltaGens updateMemDeltaGens, isDistinct bool) windowMemTest {
+	windowTest := buildWindowTester(funcName, tp, constantArg, orderByCols, numRows)
+	pt := windowMemTest{
+		windowTest:         windowTest,
+		allocMemDelta:      allocMemDelta,
+		updateMemDeltaGens: updateMemDeltaGens,
+		isDistinct:         isDistinct,
+	}
+	return pt
+}
+
+func (s *testSuite) TestMemRowNumber(c *C) {
```

Comment:
IMO, we should move this to `row_number_test.go`

Reply:
agree.